CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+* fix(lib): updated the dependencies (923f112a)
+
+* Merge pull request #304 from msenosiain/add-tabindex-attribute-toggle-visibility (9892c80f)
+* Merge pull request #318 from AndriiNeverov/schematics-angular11-support (96424510)
+* Upgraded Schematics to 11.1.2 (d5744a13)
+* Merge branch 'master' into add-tabindex-attribute-toggle-visibility (84eb5b79)
+* fix(project): upgraded angulat to v11.1.2 (920cfc2c)
+* Merge branch 'master' into add-tabindex-attribute-toggle-visibility (ad73e28e)
+* Toggle visibility: Add option input for tabindex value (ea84d761)
+
+* fix(project): updated angular and other dependencies (d9269172)
+
+* fix(project): upgraded angular cdk and angular material to v11.0.1 (a62aad5b)
+* docs(project): updated the README.md (53e69be5)
+* Merge pull request #311 from LiorCorp/feature/custom-mat-icon (7ccca45a)
+* add custom mat-icon in mat-password-strenght-info (da8d0eb7)
+* Merge pull request #297 from LakMoore/patch-1 (a7ca5718)
+* Add svgIcon attributes (32a0e3bc)
+
+* docs(project): updated the README.md (59409558)
+
+* fix(lib): solved issues related to the schematics (387cffe2)
+* feat(lib): upgraded rxjs (34f2ae81)
+* feat(lib): upgraded nguniversal to v11 (95e0ec1c)
+* fix(package): upgraded angular material to v11 (e587b64f)
+* fix(package): upgraded angular to v11 (27f9978e)
+* Merge remote-tracking branch 'origin/master' (a166ee93)
+* Merge pull request #305 from yerevin/patch-1 (b9d5ee3c)
+* Update to Angular10 (0b69e049)
+
 * fix(package): updated the peer dependencies of the library (a1286312)
 * fix(package): minor (9734ea12)
 * fix(package): updated @nguniversal/express-engine (6602952b)
```

Comment:
please can you remove your changes here because it will be automatically generated ?